org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.gradle/org.xtext.example.gradle.parent/gradle/source-layout.gradle

```diff
@@ -15,6 +15,9 @@ jar {
 	from('model') {
 		into('model')
 	}
+	from(sourceSets.main.allSource) {
+		include '**/*.xtext'
+	}
```

Comment:
yes but in maven layout we have src/main/java with xtext+mwe2 and src/main/resources with nothing